### PR TITLE
Performance improvements

### LINKED ISSUE #2458: Improve Nova performance
https://github.com/laravel/nova-issues/issues/2663

### LINKED ISSUE #2468: Look at optimizing dashboard load
From Sentry data it looks like a lot of queries are being run more than once. Cache results to reduce load time.

### LINKED ISSUE #2479: Add link headers to get browsers fetching assets as early as possible

### PR DESCRIPTION
**Pull request by kberzinch**
Fixes #2468
Fixes #2458

### CODE REVIEW

File: app/Models/User.php [hunk 1]
```diff
@@ -802,7 +802,7 @@ public function getHomeDepartmentAttribute(): ?string
     {
         $uid = $this->uid;
 
-        return Cache::remember('home_department_'.$uid, now(), static function () use ($uid): ?string {
+        return Cache::remember('home_department_'.$uid, now()->addDay(), static function () use ($uid): ?string {
```
Comment (JoshOldenburg) [inline app/Models/User.php:805]: Nice

File: app/Providers/NovaServiceProvider.php [hunk 2]
```diff
@@ -72,6 +73,11 @@ protected function gate(): void
      */
     protected function cards(): array
     {
+        if ('/nova-api/dashboards/main' !== $_SERVER['REQUEST_URI'] &&
+            ! Str::startsWith($_SERVER['REQUEST_URI'], '/nova-api/metrics/')) {
```
Comment (JoshOldenburg) [inline app/Providers/NovaServiceProvider.php:77]: There's not a Laravel way to get that variable?

**Review by JoshOldenburg — APPROVED** [review]

File: app/Providers/NovaServiceProvider.php (hunk 2 above)
Comment (kberzinch) [reply app/Providers/NovaServiceProvider.php:77]: Maybe, let me try something

File: app/Models/User.php (hunk 1 above)
Comment (evan10s) [inline app/Models/User.php:805]: It seems like the second parameter of this function, `now()...`, is the duration to store the item in the cache if the item isn't present.  Unless you can give it a date, but that's not what it looks like based on how [the docs](https://laravel.com/docs/8.x/cache#retrieve-store) describe this function.  How I'm reading it, it seems like you'd want `24*60*60` for the seconds value.

File: config/htmlmin.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    |--------------------------------------------------------------------------
+    | Ignore Blade Files
+    |--------------------------------------------------------------------------
+    |
+    | Here you can specify paths, which you don't want to minify.
+    |
+    */
+
+    'ignore' => [
+        'resources/views/emails',
```
Comment (evan10s) [inline config/htmlmin.php:62]: Why can't we minify these things?

Comment (kberzinch) [reply config/htmlmin.php:62]: That was just the default config. I don't think it matters much either way.

File: app/Models/User.php (hunk 1 above)
Comment (kberzinch) [reply app/Models/User.php:805]: The [source](https://github.com/laravel/framework/blob/9.x/src/Illuminate/Support/Facades/Cache.php#L22) indicates that a `\DateTimeInterface` can be provided in that field.

**Review by evan10s — APPROVED** [review]

**Review by JoshOldenburg — APPROVED** [review]

**Pull request merged**